packages/react-native/README.md

```diff
@@ -18,6 +18,63 @@ $ npm install @dashx/react-native
 $ yarn add @dashx/react-native
 ```
 
+### Setup for Android
+
+DashX requires Google Services installed in your app in order to Firebase to work.
+
+To install Google Services:
+
+- Add `google-services` plugin in your `/android/build.gradle`
+
+```gradle
+buildscript {
+  dependencies {
+    // ... other dependencies
+    classpath 'com.google.gms:google-services:4.3.3'
+  }
+}
+```
+
+- And then add this line in your `/android/app/build.gradle`
+
+```gradle
+apply plugin: 'com.google.gms.google-services'
+```
+
+- Add your Android app on Firebase Console.
+
+- Download `google-services.json` from there.
+
+- Add `google-services.json` at the following location `/android/app/google-services.json`
+
+### Setup for ios
+
+- At the top of the file `/ios/{projectName}/AppDelegate.m` import Firebase:
+
+```objective-c
+#import <Firebase.h>
+```
+
+- In the same file, inside your `didFinishLaunchingWithOptions` add this:
+
+```objective-c
+if ([FIRApp defaultApp] == nil) {
+  [FIRApp configure];
+}
+```
+
+- In your pod file add this:
+
+```ruby
+pod 'FirebaseInstanceID', :modular_headers => true
+```
+
+- Add your ios app on Firebase Console.
```

Comment:
Fix `iOS` capitalization here as well. Add link to Firebase Console if possible.